src/read.c

```diff
@@ -4593,6 +4597,34 @@ static avifResult avifParseMinimizedImageBox(avifDecoderData * data,
     }
     AVIF_ASSERT_OR_RETURN(meta->properties.count == 29);
 
+    // ISO/IEC 23008-12 Section 6.5.1:
+    //   Readers shall allow and ignore descriptive properties following the first transformative or unrecognized
+    //   property, whichever is earlier, in the sequence associating properties with an item.
+    //   Writers should arrange the descriptive properties specified in 6.5 prior to any other properties in the
+    //   sequence associating properties with an item.
+    //
+    // irot and imir are transformative properties, so associate them last.
```

Comment:
Optional: When I read this comment, I was wondering why we didn't seem to be following the requirement on readers (this code is in src/read.c). Later I realized that we are simulating writing a meta box here. It would be good to clarify this. One possibility might be to remove the requirement on readers (lines 4601-4602).

Reply:
Done in https://github.com/AOMediaCodec/libavif/pull/2839